frontend/web/styles/3rdParty/_hljs.scss

```
@@ -83,9 +83,10 @@
   overflow-x: auto;
   @include customScrollDark();
   padding: 0.5em;
-  background: $code-bg;
+  // Theme-adaptive surface: light in light mode, dark in dark, via tokens.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)